docs/local-setup/running-testnet.md

```diff
@@ -103,24 +84,36 @@ apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev libibe
 Then run the script:
 
 ```bash
-git clone https://github.com/nearprotocol/nearcore.git
+git clone {-b <branch name from where you want to clone>} https://github.com/nearprotocol/nearcore.git
```

Comment:
confusing variant in command -- please break this out into a single recommended command or include both with a note re: how to choose one over the other

Reply:
Changed it to single recommended command for BetaNet. Instrcutions added right after to clone for TestNet.